lib/fastlane/actions/podio_item.rb

```diff
@@ -0,0 +1,205 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      PODIO_ITEM_URL = :PODIO_ITEM_URL
+    end
+
+    class PodioItemAction < Action
+      AUTH_URL = 'https://podio.com/oauth/token'
+      BASE_URL = 'https://api.podio.com'
+
+      def self.run(params)
+        require 'rest_client'
+        require 'json'
+        require 'uri'
+
+        post_item(params)
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        'Creates an item within your Podio app. In case an item with the given identifying value already exists within your Podio app, it updates that item. See https://developers.podio.com'
+      end
+
+      def self.details
+        "Use this action to create or update an item within your Podio app
+        (see https://help.podio.com/hc/en-us/articles/201019278-Creating-apps-).
+        Pass in dictionary with field keys and their values.
+        Field key is located under Modify app -> Advanced -> Developer -> External ID
+        (see https://developers.podio.com/examples/items).
+        Have a look at http://??? how you can leverage from using Podio and Fastlane."
```

Comment:
I am working on a blog post where I want to describe how to use Podio & Fastlane & Jenkins to automate app delivery. This was supposed to link user of this action to that blog post (not published yet). I temporarily removed that until the blog post goes live.